### PR TITLE
GUVNOR-3483: [Guided Decision Tree]: Unable to add field in the update node

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3483

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/popups/ActionFieldValueEditor.java [hunk 1]
```diff
@@ -84,85 +79,82 @@
...
+                                           return;
+                                       }
+
+                                       //Add them to the ListBox
+                                       int selectedIndex = -1;
+                                       for (ModelField modelField : modelFields) {
+                                           final String fieldName = modelField.getName();
+                                           if (!fieldName.equals(DataType.TYPE_THIS)) {
+                                               fieldListBox.addItem(fieldName);
+                                               if (fieldName.equals(afv.getFieldName())) {
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/popups/ActionFieldValueEditor.java]: This is the only real change. It used to be ```afv.getFieldName().equals( fieldName )``` but ```afv.getFieldName()``` might have been ```null```. I've reversed the comparison so the LHS is always a known non-```null``` value.

**Comment by manstis** [comment]
@Rikkola Looks like your "operators" PR has broken ```drools-wb``` :-(

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by Rikkola** [comment]
Improved test coverage too. Good good.

**Comment by manstis** [comment]
Hi @jomarko you mentioned the other day you were "all things _guided_". Did you want to review this?

**Comment by jomarko** [comment]
@manstis thanks, I am interested definitely, but at first, please, see my email about this.

**Comment by manstis** [comment]
@jomarko I've replied to your email...

File: drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/popups/ActionFieldValueEditor.java (hunk 1 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/popups/ActionFieldValueEditor.java]: Hehe, answer for question I wanted to put into review comment.

**Pull request merged**